data/supported-devices/devices/xiaomi_onclite.yml

```diff
@@ -0,0 +1,181 @@
+manufacturer: Xiaomi
```

Comment:
Please add a template file instead, Please refer readme file for updated instructions.